src/EditorFeatures/Core/Extensibility/NavigationBar/NavigationBarItems/RoslynNavigationBarItemKind.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Editor.Extensibility.NavigationBar
+{
+    internal enum RoslynNavigationBarItemKind
+    {
+        Symbol,
+        GenerateDefaultConstructor,
+        GenerateEventHandler,
+        GenerateFinalizer,
+        GenerateMethod,
+    }
```

Comment:
Just asking to understand future intent: what is this being used for? I assume it's for something other than just the type switching in the generate method?